docs/developer/Releases.md

```diff
@@ -0,0 +1,116 @@
+Release Approach
+================
+
+## Table of Contents
+
+* [Versioning](#versioning)
+  * [API Compatibility](#api-compatibility)
+  * [Modules Providing API](#modules-providing-api)
+  * [Towards a First Release](#towards-a-first-release)
+* [Legal Documentation Requirements](#legal-documentation-requirements)
+  * [License and Notice Files](#license-and-notice-files)
+  * [Creating the Notice File](#creating-the-notice-file)
+  * [Background Information](#background-information)
+* [Publishing Maven Artifacts](#publishing-maven-artifacts)
+  * [Naming Convention](#naming-convention)
+
+### Versioning
+
+The Eclipse Dataspace Connector employs [SemVer](https://semver.org) for versioning and distinguishes between the following releases as defined by the [Eclipse Handbook](https://www.eclipse.org/projects/handbook/#release):
```

Comment:
I would say "will employ" so we don't give anyone the false impression this will happen before 1.0 (even though it is mentioned below).